### PR TITLE
add: alert components

### PR DESCRIPTION
**Pull request by Varu98**

### CODE REVIEW

**Comment by Varu98** [comment]
This is the preview link for the alert component

https://deploy-preview-7--bee-ui.netlify.app/pages/alert.html

File: styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+.alert-info-wrapper .msg {
+  margin-right: 1rem;
+  font-size: 1rem;
+  font-weight: 400;
+  color: #1c5796;
+}
+.alert-info-wrapper .fa-times {
+  font-size: 1rem;
+  color: #1c5796;
```
Comment (manojasarsa) [inline styles/alert.css:96]: You can use color varibales for colors.

**Review by manojasarsa — APPROVED** [review]
Rest it looks good! 👍

File: styles/alert.css (hunk 1 above)
Comment (Varu98) [reply styles/alert.css:96]: Sure will give it a spin, Thanks!

File: styles/alert.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+  justify-content: space-between;
+  max-height: 3rem;
+  align-items: center;
+}
+
+.alert-warning-wrapper .fa-exclamation-circle {
+  margin: 0 0.5rem;
+  font-size: 1rem;
+  color: #ce8500;
```
Comment (srishti-maurya) [inline styles/alert.css:22]: may be you may use color variables here

**Review by srishti-maurya — APPROVED** [review]
looks good

File: styles/alert.css (hunk 2 above)
Comment (Varu98) [reply styles/alert.css:22]: will check them out Thanks !

**Pull request merged**